test/integration/patients/sidebar/action-sidebar.js

```diff
@@ -712,6 +718,158 @@ context('action sidebar', function() {
       .should('contain', 'The Action you requested does not exist');
   });
 
+  specify('action phone dialer', function() {
```

Comment:
Quick summary of test:

1. Open dialer picklist and verify phone numbers are shown properly and are ordered based on `preferred` value.
2. Close dialer picklist and open it again. Verify phone numbers are displayed via cached data. Api request shouldn't be made again.
3. Change action to done and verify dialer button is set to `disabled`.
4. Change parent flow to done and verify dialer button is set to `disabled`.

Reply:
Potentially move some of this to a component test on the `dialer_component.js`.